src/main/java/org/rug/data/project/GitVersion.java

```diff
@@ -49,10 +51,14 @@ public synchronized SourceCodeRetriever getSourceCodeRetriever() {
     public String parseVersionString(Path f) {
         var fileName = f.getFileName().toString();
         int endIndex = f.toFile().isDirectory() ? fileName.length() : fileName.lastIndexOf('.');
-        var splits = fileName.substring(0, endIndex).split("-");
-        setVersionIndex(Long.parseLong(splits[1]));
-        versionDate = String.join("-", splits[2].split("_"));
-        commitName = splits[3];
+        var splits = new ArrayList<>(Arrays.asList(fileName.substring(0, endIndex).split("-")));
+        if (splits.get(1) .equals("file")) {
+            // Arcan for singleVersion Java adds the word "file" to the outputted GraphML file
+            splits.remove(1);
```

Comment:
This check shouldn't be necessary, as I changed arcan to output the file names consistently across single-version and history analysis modes.

Reply:
Rolled back the changes!